tests/env/express/public/headers.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+</head>
+
+<body>
+  <div id="content"></div>
+</body>
+
+</html>
+<script src="./meilisearch.umd.js"></script>
+<script>
+  ; (async () => {
+
+    const errorDiv = document.createElement("div");
+    errorDiv.setAttribute("id", "error");
+
+    let error = 'NO ERRORS'
+    try {
+      const task = await client.createIndex(UID)
+      await client.waitForTask(task.uid)
+      await fetch('http://localhost:7700/indexes/movies/documents', {
+        method: 'POST',
+        headers: {
+          Authorization: `Bearer masterKey`,
+          'User-Agent': 'plif plouf',
```

Comment:
It's on purpose you let `plif plouf` ?

Reply:
Yes, hahaha its cute 😊 the content is not very important